assets/js/custom/MyProfile.js

```diff
@@ -135,18 +135,66 @@ let MyProfile = function (profile_url, save_username,
     $.get(self.toggle_visibility_url + '/' + id, {}, function (data) {
       let visibilityLockId = $('#visibility-lock-' + id)
       let visibilityLockOpenId = $('#visibility-lock-open-' + id)
+      let programName = $('#program-' + id).find('.program-name').text()
+  
+      let translations = []
+      translations.push({
+        key  : '%programName%',
+        value: programName
+      })
+     
+      
```

Comment:
there are quite many new lines for no real reason in your added code. Please change that. E.g. 146, 160-162, 177-178, ...
would be best to check the code style again. (indentions, ..)